docs/API_ref.md

```diff
@@ -0,0 +1,24 @@
+<h1>API References</h1>
+
+| Property   |      Type     |  Description |
+|:-----------|:--------------|:-------------|
+| backupImages | bool |      |
```

Comment:
`backupImages` is a boolean to determine whether we want our velero install to backup internal images when an imagestream is backed up.